python/prophet/forecaster.py

```diff
@@ -53,6 +53,7 @@ class Prophet(object):
         optionally can have a column prior_scale specifying the prior scale for
         that holiday.
     seasonality_mode: 'additive' (default) or 'multiplicative'.
+    holiday_mode: 'additive' or 'multiplicative'. Defaults to seasonality_mode.
```

Comment:
Sorry a small nit: could we change the name to `holidays_mode`, to be consistent with the other holidays parameters (holidays, holidays_prior_scale). Thank you!

Reply:
Makes sense. Done!